lib/chef-dk/command/verify.rb

```diff
@@ -75,25 +75,37 @@ def components
       #
       add_component "berkshelf" do |c|
         c.gem_base_dir = "berkshelf"
-        # For berks the real command to run is "bundle exec thor spec:ci"
+        # For berks the real command to run is "#{bin("bundle")} exec thor spec:ci"
         # We can't run it right now since graphviz specs are included in the
         # test suite by default. We will be able to switch to that command when/if
         # Graphviz is added to omnibus.
-        c.unit_test { sh("bundle exec rspec --color --format progress spec/unit --tag ~graphviz") }
-        c.integration_test { sh("bundle exec cucumber --color --format progress --tags ~@no_run --tags ~@spawn --tags ~@graphviz --strict") }
+        c.unit_test do
+          sh("#{bin("bundle")} install")
```

Comment:
Why do we need to bundle install all these tests now?  I thought because we have ensured that our Gemfile bundle resolved we would have 1 copy of every necessary gem.